changes/70.changed.md

```diff
@@ -0,0 +1 @@
+Adapt use of `requests` to ["Automatic Retries" example](https://docs.python-requests.org/en/latest/user/advanced/#example-automatic-retries).
```

Comment:
I would only mention this in the changelog once and remove the fragment from the other PR.